native/audio-host/mac/verify.sh

```
@@ -0,0 +1,129 @@
+#      instead of going permanently silent on a dead object.
+#   4. Isolation   - a target that never plays yields silence while another
+#      process is audibly playing, i.e. the tap did not widen to everything.
+#
+# The audio assertions need the "System Audio Recording Only" grant, which TCC
+# attributes to the *responsible* process - here the terminal, not this script.
+# Without it every sample is zero by design; the script detects that, keeps the
+# structural assertions and skips the ones it cannot judge.
+#
+# Expected output ends with VERIFY OK.
+set -uo pipefail
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)